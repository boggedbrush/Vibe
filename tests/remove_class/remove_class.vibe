# VibeSpec: 1.4
patch_type: remove_class
file: hello.py
name: OldClass
