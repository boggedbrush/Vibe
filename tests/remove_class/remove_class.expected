# hello.py
def greet(name):
    print(f"Hello, {name}!)  # notice missing paren?

class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')
